client/lib/cart-values/cart-items.js

```diff
@@ -683,6 +683,19 @@ export function fillGoogleAppsRegistrationData( cart, registrationData ) {
 	);
 }
 
+const getDomainPartFromEmail = emailAddress => {
+	emailAddress = emailAddress || '';
+	return emailAddress.replace( /.*@([^@]+)$/, '$1' );
+};
+
+export function needsExplicitAlternateEmailForGSuite( cart, contactDetails ) {
+	return some( cart.products, [ 'meta', getDomainPartFromEmail( contactDetails.email ) ] );
```

Comment:
Shouldn't we display the `Alternate Email` field in the case there is no email address in the contact details?

Reply:
Yes - you're right!